src/api/python/CMakeLists.txt

```diff
@@ -113,6 +113,17 @@ set_target_properties(cvc5_python_base
 
 python_extension_module(cvc5_python_base)
 
+find_package(CVC5PythonicAPI)
+
+add_custom_target(
```

Comment:
Add comment about why we have to copy the directory. Can't we directly install the API to `"${CMAKE_CURRENT_BINARY_DIR}/cvc5/pythonic"` in `ExternalProject_Add`?